project/src/test/puzzle/piece/test-piece-kicks.gd

```diff
@@ -108,6 +126,8 @@ func _create_active_piece(ascii_grid: Array) -> ActivePiece:
 	
 	var _active_piece: ActivePiece
 	for pos_arr_index in range(piece_type.pos_arr.size()):
+		if forced_piece_orientation != -1 and pos_arr_index != forced_piece_orientation:
+			continue
```

Comment:
This could be streamlined by having the loop only loop over a single orientation.

Reply:
Improved for loop.